server/WebAPI/Services/ResetPasswordService.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using WebAPI.DTO;
+using WebAPI.Interfaces;
+
+namespace WebAPI.Services
+{
+  public class ResetPasswordService
```

Comment:
its kind of duplicate of ResetPasswordService higher in PR, but this one with few messages

Reply:
it was deleted